samples/qa-a11y/src/views/ArcSlider.js

```diff
@@ -0,0 +1,26 @@
+import ArcSlider from '@enact/agate/ArcSlider';
+import React from 'react';
+
+import Section from '../components/Section';
+
+import appCss from '../App/App.module.less';
+
+const ArcSliderView = () => (
+	<>
+		<Section title="Default">
+			<ArcSlider alt="Normal" />
+			<ArcSlider alt="With slotCenter" slotCenter="SlotCenter" />
```

Comment:
SlotCenter looks overlapped with the slider. Would you please replace this with something else?